obspy/io/ah/core.py

```diff
@@ -382,12 +382,34 @@ def _pack_trace_wout_ah_dict(tr, packer, codesize, chansize,
     packer.pack_float(0)
 
     # Poles and Zeros are not provided by stream object, are set to 0
-    for _i in range(0, 30):
-        packer.pack_float(0)
-        packer.pack_float(0)
+    if hasattr(tr.stats, 'response'):
+        poles, zeros = tr.stats.response.get_paz()
```

Comment:
In general we nowadays don't encourage "magic uses" of additional items in `Trace.stats` anymore.. but if it's important for you we might still accept this change I guess..